lib/pages/chat/chat.dart

```diff
@@ -126,6 +125,9 @@ class ChatController extends State<Chat>
           EventTypes.Sticker,
           EventTypes.Encrypted,
           EventTypes.CallInvite,
+          EventTypes.RoomAvatar,
+          EventTypes.RoomName,
+          EventTypes.RoomTopic,
```

Comment:
maybe side effect for the case of creating chat?

Reply:
wrong in the view of group creator. 

<img width="1278" alt="image" src="https://github.com/linagora/twake-on-matrix/assets/6462404/69b25c6a-fd8e-4a86-9949-75ea20306654">